docs/02.API-REFERENCE.md

```diff
@@ -952,6 +952,29 @@ typedef void (*jerry_module_state_changed_callback_t) (jerry_module_state_t new_
 **See also**
 - [jerry_module_set_state_changed_callback](#jerry_module_set_state_changed_callback)
 
+## jerry_module_import_meta_callback_t
+
+**Summary**
+
+Callback which is called when an import.meta expression of a module is evaluated the first time.
```

Comment:
I think we need a bit of comment to describe that the user/developer should add properties to the `meta_object` to "configure" the `import.meta` value.